tests/e2e/test_gateway_commands.py

```diff
@@ -196,5 +197,38 @@ def test_deploy_account_is_available(protostar: ProtostarFixture):
     )
 
 
+@pytest.mark.usefixtures("init")
+def test_calculate_account_address_is_available(protostar: ProtostarFixture):
+    def run(json_format: bool):
+        optionals = []
+        if json_format:
+            optionals.append("--json")
+        return protostar(
+            [
+                "--no-color",
+                "calculate-account-address",
+                "--account-class-hash",
+                "1",
+                "--account-address-salt",
+                "1",
+                *optionals,
+            ],
+            ignore_stderr=True,
+        )
+
+    human_output = run(json_format=False)
+    json_output = run(json_format=True)
+
+    assert (
+        "Address: 0x033f7162354afe9442cc91d8f62a09613d33558c9fcdaf8a97912895e3f7ce93\n"
+        == human_output
+    )
+
+    assert (
+        json.loads(json_output)["address"]
+        == "0x033f7162354afe9442cc91d8f62a09613d33558c9fcdaf8a97912895e3f7ce93"
+    )
```

Comment:
```suggestion
    assert json.loads(json_output) == {
        "address": "0x033f7162354afe9442cc91d8f62a09613d33558c9fcdaf8a97912895e3f7ce93"
    }
```

nitpick: such assertion would be more strict

Reply:
and fragile